node/service/lib/device.d.ts

```diff
@@ -24,9 +24,14 @@ declare namespace Device {
         secondaryKey: string;
     }
 
+    interface _X509Thumbprint {
+        primaryThumbprint: string;
+        secondaryThumbprint: string;
+    }
+
     interface Authentication {
-        SymmetricKey?: _SymmetricKey;
-        x509Thumbprint?: string;
+        symmetricKey?: _SymmetricKey;
```

Comment:
SymmetricKey needs to stay uppercase otherwise it's a breaking change of the API and needs to be reflected in javascript too. This was a typo in the first version of the SDK that will be fixed in the next major release